gammapy/irf/psf/map.py

```diff
@@ -270,10 +272,11 @@ def get_psf_kernel(
             center position is used.
         max_radius : `~astropy.coordinates.Angle`, optional
             Maximum angular size of the kernel map.
+            Default is None and it will be computed for the `containment` fraction set.
         containment : float, optional
-            Containment fraction to use as size of the kernel. The max. radius
-            across all energies is used. The radius can be overwritten using
-            the `max_radius` argument. Default is 0.999.
+            Containment fraction to use as size of the kernel.
+            The radius can be overwritten using the `max_radius` argument.
+            Default is 0.999.
         factor : int, optional
```

Comment:
If you plan to deprecate `factor` you should add the proper decorator.

Reply:
it is already deprecated since v1.2